Marlin/Configuration.h

```diff
@@ -536,20 +536,23 @@
 // @section probes
 
 //
-// See http://marlinfw.org/configuration/probes.html
+// IT IS HIGHLY RECOMMENDED TO VISIT THE PROBES SECTION OF THE MARLIN WIKI
+// AT http://marlinfw.org/configuration/probes.html
 //
 
 /**
  * Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
  *
- * Enable this option for a probe connected to the Z Min endstop pin.
+ * Use this option if you want to use the probe for both homing and probing. The
+ * probe must be connected to the Z Min endstop pin.
  */
 #define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
```

Comment:
The existing note is accurate. I didn't want to strongly associate this with using the probe for homing, since that comes along for free. This option also applies to any machine that homes to Z_MAX, so that Z_MIN is used as the probe pin — so it generally applies to all Deltas — even though it does not enable homing with the probe.